<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formmouklien</class>
 <widget class="QWidget" name="Formmouklien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nomouLabel">
      <property name="text">
       <string>nomou</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomouLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="idklienLabel">
      <property name="text">
       <string>idklien</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idklienLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jangkawaktuLabel">
      <property name="text">
       <string>jangkawaktu</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="jangkawaktuLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="tglmouDateEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tglmouLabel_2">
      <property name="text">
       <string>tglmou</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
